web/webpack.config.js

```diff
@@ -145,19 +146,26 @@ module.exports = composePlugins(withNx({
       });
     }
 
-    if (rule.test.toString().includes('styl')) {
-      const r = rule.oneOf.filter((r) => r.use && r.use.find((u) => u.loader && u.loader.includes('stylus-loader')));
+    if (rule.test.toString().match(/css|scss|sass|styl/)) {
+      const r = rule.oneOf.filter((r) => r.use && r.use.find((u) => u.loader && u.loader.includes('css-loader')));
 
-      r.forEach(_r => {
-        const l = _r.use.filter((u) => u.loader && u.loader.includes('stylus-loader'));
+      r.forEach((_r) => {
         const cssLoader = _r.use.find(use => use.loader && use.loader.includes('css-loader'));
 
         if (cssLoader && cssLoader.options) {
           cssLoader.options.modules = {
             localIdentName: css_prefix + '[local]', // Customize this format
           };
         }
+        console.log(_r, cssLoader)
```

Comment:
another one. plus you don't use `log` you imported earlier

Reply:
yeah, gotta cover that under the linting strategy initiative